presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -306,6 +316,24 @@ private ColumnMapping jsonColumnMapping()
                 DISABLE_PUSHDOWN);
     }
 
+    private static Optional<ColumnMapping> getUnsignedMapping(JdbcTypeHandle typeHandle)
+    {
+        if (typeHandle.getJdbcTypeName().isPresent() &&
+            typeHandle.getJdbcTypeName().get().toUpperCase(ENGLISH).contains("UNSIGNED")) {
```

Comment:
Does mysql support user defined types?

I think it would be better to have a (case-insensitive?) equality condition on type name rather than contains check

```
if (jdbcTypeName.equalsIgnoreCase("tinyint unsigned")) {
```

-- like we do for `json`